### PR TITLE
FFM-7741: Adds git reset procedure

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to preview**](https://646389f61b60391bec5e4f2d--harness-developer.netlify.app/docs/feature-flags/manage-featureflags-in-git-repos#reset-the-git-connection)

### CODE REVIEW

File: docs/feature-flags/manage-featureflags-in-git-repos.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Manage flags using Git Experience
+title: Manage flags with Git
```
Comment (brian-f-harness) [inline docs/feature-flags/manage-featureflags-in-git-repos.md:2]: May want to consider leaving Experience in the title and using `sidebar_label` to maintain SEO if trying to shorten for the left nav

File: docs/feature-flags/manage-featureflags-in-git-repos.md [hunk 2]
```diff
@@ -236,11 +236,11 @@ If you want to always commit to the same branch, and not be prompted for commit
 
 ## Turn off syncing with Git
 
-To turn off syncing with Git:
+When you turn off syncing with Git, any changes you make to flags in Harness are not committed to the flags YAML file in your Git repository, and any changes to that file in your repository are not synced with Harness, until you [turn on syncing](#turn-on-syncing-with-git) again.
```
Comment (brian-f-harness) [inline docs/feature-flags/manage-featureflags-in-git-repos.md:239]:
```suggestion
When you turn off syncing with Git, any changes you make to flags in Harness are not committed to the flags YAML file in your Git repository. Any changes to that file in your repository are not synced with Harness until you [turn on syncing](#turn-on-syncing-with-git) again.
```

File: docs/feature-flags/manage-featureflags-in-git-repos.md [hunk 3]
```diff
@@ -252,7 +252,15 @@ To turn off syncing with Git:
   <img src={git_8} alt="A screenshot of the Git sync toggle turned off." height="500" width="300" />
   ``` 
 
-## See also
+## Reset the Git connection
+
+This procedure removes the connection you set up to your Git repository when you [set up Git Experience](#set-up-git-experience).
```
Comment (brian-f-harness) [inline docs/feature-flags/manage-featureflags-in-git-repos.md:257]:
```suggestion
This procedure removes the connection to your Git repository when you [set up Git Experience](#set-up-git-experience).
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm. Left a couple small suggestions.

**Review by Kira-Jennings — APPROVED** [review]
Looks good Gigi!

**Comment by StrawberryPad** [comment]
Thanks @Kira-Jennings !

**Pull request merged**